lib/cisco_node_utils/cmd_ref/vpc.yaml

```diff
@@ -55,20 +57,19 @@ dual_active_exclude_interface_vlan_bridge_domain:
     set_value: "<state> dual-active exclude interface-vlan <range>"
   default_value: 'none'
 
-#vPC+ CLIs
-fabricpath_multicast_load_balance: 
-  kind: boolean
-  get_value: '/^fabricpath multicast load-balance'
-  set_value: "<state> fabricpath multicast load-balance"
-  default_value: false
-
-fabricpath_switch_id: 
+fabricpath_emulated_switch_id: 
+  _exclude: [N3k, N5k, N6k, N9k]
   kind: int
   get_value: '/^fabricpath switch\-id\s+(\d+)/'
-  set_value: "<state> fabricpath switch-id <swid>"
+  set_value: '<state> fabricpath switch-id <swid>'
   default_value: false
 
-#END vPC+ CLIs
+fabricpath_multicast_load_balance: 
+  _exclude: [N3k, N5k, N6k, N9k]
+  kind: boolean
+  get_value: '/^fabricpath multicast load\-balance/'
```

Comment:
Anchoring it protects us against any unexpected additions to the CLI (say 'fabric-path multicast load-balance *disable*') - if there are cases where we *know* there are trailing spaces, we can explicitly handle them in the regex, but in general this shouldn't be needed.

In Ruby regexp `-` is only significant inside a `[...]` block - so this isn't necessary, although harmless.